.github/workflows/pull-integration.yaml

```diff
@@ -18,7 +18,6 @@ on:
       - ".github/**" #Ignoring since build job isn't triggered on the workflow directory changes
     branches:
       - main
-      - 'release-**'
```

Comment:
Why is this removed?

Reply:
PR provides another workflow for pull requests to the release branch